docs/export-to-csv.md

```diff
@@ -0,0 +1,59 @@
+# Export ADOs to CSV Action
+
+Archipelago has a specialty Action, labeled `Export Archipelago Digital Objects to CSV content item`, which enables you to generate a CSV file for a selected group of Archipelago Digital Objects (ADOs). 
+
+## Where to Find
+
+In default Archipelago deployments, this speciality action is found in the `Action` menu for the main Content page:
+
+- Accessed through the `Content` menu item
```

Comment:
@alliomeria is this right? admin/content?

Reply:
(just asking, I did not remember we had it there too! so cool)

Reply:
I believe so yes, in latest deployments, available by default